bin/update_folders.py

```diff
@@ -105,6 +106,13 @@ def update_folders(
             f"Non-empty prediction and log files must be provided to update folders for submission {submission_id}. Exiting."
         )
 
+    # Make a record of the file entity's Synapse ID so it can be stored as an annotation for the given submission
+    file_synid = file_entity.id
+    output_annotation = {f"{folder_name}_id": file_synid}
+
+    with open("output_annotation.json", "w") as o:
+        o.write(json.dumps(output_annotation))
```

Comment:
This overwrite `output_annotations.json` created for predictions, so we only get the `docker_logs_id` annotation. Change this functionality such that it _appends_ to the content already in `output_annotation.json` (even if empty).